.github/workflows/pull-request-build.yml

```diff
@@ -60,5 +62,7 @@ jobs:
           dockerfile: "packages/client/Dockerfile"
           image_name: "${{ vars.DOCKER_USERNAME }}/client"
           ## Tag as pr-<pr-id>
-          tag: "pr-${{ github.event.pull_request.number }}"
+          tags: |
+            "pr-${{ github.event.pull_request.number }}"
+            "extra-tag-to-test"
```

Comment:
This second tag will be removed here... this is to test that support for multiple tags works before merging

Reply:
Removed